jkube-kit/generator/api/src/test/java/org/eclipse/jkube/generator/api/FromSelectorTest.java

```diff
@@ -31,13 +31,10 @@
 
 public class FromSelectorTest {
 
-  @Mocked
   JavaProject project;
 
-  @Mocked
   Plugin plugin;
 
-  @Mocked
   KitLogger logger;
```

Comment:
These variables are null, meaning they are useless.
Please remove them, and their usages (L57, L59)